extensions/cta/src/cta.tsx

```diff
@@ -0,0 +1,858 @@
+    if (!lastUpdate) return "";
+    return `Updated ${lastUpdate.toLocaleTimeString([], { hour: "numeric", minute: "2-digit" })}`;
+  };
+
+  const renderAccessories = (prediction: BusPrediction | TrainArrival) => {
+    if (transitType === "bus") {
+      const busPrediction = prediction as BusPrediction;
+      const isDue = busPrediction.prdctdn === "DUE";
+      const minutes = isDue ? 0 : parseInt(busPrediction.prdctdn);
+
+      return [
+        {
+          text: {
+            color: isDue ? Color.Red : minutes <= 5 ? Color.Red : minutes <= 10 ? Color.Orange : Color.Green,
+            value: isDue ? "Due" : `${minutes} min`,
+          },
+          tooltip: `Arriving at ${busPrediction.prdtm}`,
+        },
+      ];
+    } else {
+      const trainPrediction = prediction as TrainArrival;
+      try {
+        const [datePart, timePart] = trainPrediction.arrT.split("T");
+        const [year, month, day] = datePart.split("-").map(Number);
+        const [hours, minutes, seconds] = timePart.split(":").map(Number);
+
+        const arrivalTimestamp = Date.UTC(year, month - 1, day, hours + 6, minutes, seconds);
+        const now = Date.now();
+
+        const minutesUntil = Math.round((arrivalTimestamp - now) / 60000);
+        const isDelayed = trainPrediction.isDly === "1";
+        const isDue = minutesUntil < 2;
+
+        // Only show green if due (< 2 minutes), otherwise use default color
+        // Always show red if delayed
+        const textColor = isDelayed ? Color.Red : isDue ? Color.Green : Color.PrimaryText;
+
+        const timeText = isDue ? "Due" : `${minutesUntil} min`;
+
+        return [
+          {
+            text: {
+              color: textColor,
+              value: `${formatTime(trainPrediction.arrT)} (${timeText})`,
+            },
+          },
+          ...(isDelayed
+            ? [
+                {
+                  text: "Delayed",
+                  tooltip: "This train is experiencing delays",
+                },
+              ]
+            : []),
+        ];
+      } catch (error) {
+        console.error("Error parsing date:", error, trainPrediction.arrT);
+        return [
+          {
+            text: {
+              color: trainPrediction.isDly === "1" ? Color.Red : Color.PrimaryText,
+              value: formatTime(trainPrediction.arrT),
+            },
+          },
+        ];
+      }
+    }
+  };
+
+  const renderActions = (prediction: BusPrediction | TrainArrival) => {
+    const stopId = transitType === "bus" ? (prediction as BusPrediction).stpid : (prediction as TrainArrival).staId;
+
+    const stopName = transitType === "bus" ? (prediction as BusPrediction).stpnm : (prediction as TrainArrival).staNm;
+
+    return (
+      <ActionPanel>
+        <ActionPanel.Section>
+          <Action
+            icon={Icon.ArrowLeft}
+            title="Go Back"
+            shortcut={{ modifiers: ["cmd"], key: "[" }}
+            onAction={() => setSearchText("")}
+          />
```

Comment:
We don't have "Go Back" actions anywhere else in Raycast. Perhaps we can allow users to use the escape key, as they are accustomed to?